packages/models/src/quickChat.ts

```diff
@@ -0,0 +1,10 @@
+export enum QuickChatOption {
+	GG = "GG",
+	HAPPY = "😃",
```

Comment:
probably good to make these images at some point for better browser support but I like the emojis 😄

Reply:
I've left this comment be as I'm not sure how I would go about changing them to images - any advice on this one ?

Reply:
well rather than just rendering the text of the quick chat to the screen, some of them could be IDs that just render a specific `<img />` element